pkgs/tools/security/genpass/default.nix

```diff
@@ -23,6 +24,7 @@ rustPlatform.buildRustPackage rec {
     CoreFoundation
     libiconv
     Security
+    zlib
```

Comment:
should just add zlib unconditionally so all builds use the system zlib.

Reply:
(And all the other dependencies are now unnecessary and do nothing, too, so it can just be `buildInputs = [ zlib ];`.)

Reply:
ah cool! will fixup - thanks for the callouts!

Reply:
should be adjusted now!